test/test_arraycontext.py

```diff
@@ -727,9 +745,51 @@ def test_norm_ord_none(actx_factory, ndim):
     norm_a_ref = np.linalg.norm(a, ord=None)
     norm_a = actx.np.linalg.norm(actx.from_numpy(a), ord=None)
 
+    if not np.isscalar(norm_a):
```

Comment:
Both array contexts should handle reduction returns consistently.

Reply:
Done(?) in https://github.com/inducer/arraycontext/pull/14/commits/95fd7be8645e6d35d71aa903f70d441facc9f0d8